app/helpers/alchemy/admin/essences_helper.rb

```diff
@@ -80,6 +80,25 @@ def essence_picture_thumbnail(content, options)
         )
       end
 
+      # Show image cropping link for given content and options?
+      def allow_image_cropping?(content, options = {})
```

Comment:
That's what I thought and tried to implement on the `Content` model, but this is the wrong way here. The `Content` model must not know anything about the essence specific stuff. 

But I could add it to the `EssencePicture`, that's a good idea.
Will provide a patch.

Thanks for pointing this out.